app/models/namespace.rb

```diff
@@ -120,6 +120,19 @@ def self.make_valid(name)
     name = final[0..MAX_NAME_LENGTH]
 
     return nil if name !~ NAME_REGEXP
+
+    # To avoid any name conflict we append an incremental number to the end
+    # of the name returns it as the name that will be used on both Namespace
+    # and Team on the User#create_personal_namespace! method
+    # TODO: workaround until we implement the namespace/team removal
+    increment = 0
+    original_name = name
+    while Namespace.exists?(name: name)
+      name = "#{original_name}#{increment}"
+      increment += 1
+      break if increment > 1000
```

Comment:
@mssola I didn't go with the `times` approach. Just added a `break if` here. Is it okay?